rfc6962-bis.xml

```diff
@@ -357,6 +357,9 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
         </t>
       </section>
       <section title="Redaction of Domain Name Labels" anchor="domain_name_redaction">
+        <t>
+          The following mechanism allows private labels to not appear in public logs, while still retaining most of the security benefits that accrue from using Certificate Transparency mechanisms. Note that TLS clients may have policies that specify additional requirements relating to the use of this redaction mechanism.
```

Comment:
Is "MAY" appropriate here?  Currently this section only uses RFC2119 keywords for requirements on CAs.  This sentence is intended to make CAs aware that they need to consider TLS client policies, so I think "may" is appropriate.

I agree with your suggestion about explicitly mentioning CT-compliance here.